<launch>
  <!-- BEGIN ROBOT 0 -->
  <group ns="robot_0">
    <param name="goal_x" type = "double" value="4" />
    <param name="goal_y" type = "double" value="4" />
    <param name="goal_theta" value="0" />
    <node pkg="multi_navigation" type="multi_navigation_send_goal3" respawn="false" name="multi_navigation_send_goal3" output="screen" args="0 4 4 0"/>
  </group>
  <!-- END ROBOT 0 -->

  <!-- BEGIN ROBOT 1 -->
  <group ns="robot_1">
    <param name="goal_x" value="5" />
    <param name="goal_y" value="5" />
    <param name="goal_theta" value="45" />
    <node pkg="multi_navigation" type="multi_navigation_send_goal3" respawn="false" name="multi_navigation_send_goal3" output="screen" args="1 5 5 0"/>
  </group>
  <!-- END ROBOT 1 -->

  <!-- BEGIN ROBOT 2 -->
  <group ns="robot_2">
    <param name="goal_x" value="6" />
    <param name="goal_y" value="6" />
    <param name="goal_theta" value="0" />
    <node pkg="multi_navigation" type="multi_navigation_send_goal3" respawn="false" name="multi_navigation_send_goal3" output="screen" args="2 6 6 0"/>
  </group>
  <!-- END ROBOT 2 -->

  <!-- BEGIN ROBOT 3 -->
  <group ns="robot_3">
    <param name="goal_x" value="7" />
    <param name="goal_y" value="7" />
    <param name="goal_theta" value="45" />
    <node pkg="multi_navigation" type="multi_navigation_send_goal3" respawn="false" name="multi_navigation_send_goal3" output="screen" args="3 7 7 0"/>
  </group>
  <!-- END ROBOT 3 -->

  <!-- BEGIN ROBOT 4 -->
  <group ns="robot_4">
    <param name="goal_x" value="8" />
    <param name="goal_y" value="8" />
    <param name="goal_theta" value="0" />
    <node pkg="multi_navigation" type="multi_navigation_send_goal3" respawn="false" name="multi_navigation_send_goal3" output="screen" args="4 8 8 0"/>
  </group>
  <!-- END ROBOT 4 -->

  <!-- BEGIN ROBOT 5 -->
  <group ns="robot_5">
    <param name="goal_x" value="9" />
    <param name="goal_y" value="9" />
    <param name="goal_theta" value="45" />
    <node pkg="multi_navigation" type="multi_navigation_send_goal3" respawn="false" name="multi_navigation_send_goal3" output="screen" args="5 9 9 0"/>
  </group>
  <!-- END ROBOT 5 -->

</launch>
